### PR TITLE
B 18492 Prime API sitAddressUpdate Deprecation Documentation Updates

### PR DESCRIPTION
**Pull request by danieljordan-caci**
## Summary

As we officially transition to having the shipment's change of `destination_address` drive the change of destination SIT service items' `sit_destination_final_address`, we need to let the Prime know this by updating the API documentation.

This PR does the following:
- updates Prime API documentation to notify them that the endpoint `createSITAddressUpdateRequest` will be deprecated 6 months from now
- in the description of the details, notified them of the change and what endpoints can be used to update the `sit_destination_final_address` after the endpoint is deprecated

### How to test

1. Fire up the client & server
2. Open up [Swagger docs](http://officelocal:3000/swagger-ui/prime.html#/sitAddressUpdate/createSITAddressUpdateRequest) to see the changes

### CODE REVIEW

File: swagger-def/prime.yaml [hunk 1]
```diff
@@ -1361,6 +1363,11 @@ paths:
         **Limitations:**
         The update can be requested for APPROVED SIT service items only.
         Only ONE request is allowed per approved SIT service item.
+
+        **DEPRECATION ON AUGUST 5TH, 2024**
+        Following deprecation, when updating a service item's final delivery address, you will need to update the shipment's destination address. This will update the destination SIT service items' final delivery address upon approval.
+        For `APPROVED` shipments, you can use [updateShipmentDestinationAddress](#mtoShipment/updateShipmentDestinationAddress)
+        For shipments in any other status, you can use [updateShipmentAddress](#mtoShipment/updateShipmentAddress)
```
Comment (traskowskycaci) [inline swagger-def/prime.yaml:1370]: Can you change `updateShipmentAddress` to `updateMTOShipmentAddress`?

Comment (danieljordan-caci) [reply swagger-def/prime.yaml]: Oops! Fixed!

**Review by traskowskycaci — APPROVED** [review]
Looks gooood!

**Comment by danieljordan-caci** [comment]
> Looks great! Just one thing, I believe now would be a good time to get in the groove of step 5 [here](https://transcom.github.io/mymove-docs/docs/api/guides/how-to-deprecate-endpoints#deprecate). If we don't we'll definitely forget about actually deprecating this!
> 
> Or in the future we could actually implement a time based system to auto deprecate which would be much more desirable, but hey this is what we have for now.
> 
> Requesting: A future ticket to exist in the backlog prior to approval

Re-requesting your review because this action is in process.

**Review by cameroncaci — APPROVED** [review]

**Pull request merged**